galaxychop/core.py

```diff
@@ -491,6 +491,12 @@ def jcirc(self, bin0=0.05, bin1=0.005):
         (neg,) = np.where(E_tot <= 0.0)
         (neg_star,) = np.where(Etot_s <= 0.0)
 
+        if len(neg) == 0:
+            raise ValueError("All Energy values are positive")
+
+        if len(neg_star) == 0:
+            raise ValueError("All Stellar Energy values are positive")
+
         # Remove the particles with E = -inf.
         (fin,) = np.where(E_tot[neg] != -np.inf)
```

Comment:
Here you can implement this function: `np.isinf` or `~np.isinf` for the opossite function.
Check doc https://numpy.org/doc/stable/reference/generated/numpy.isinf.html